boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst

```diff
@@ -0,0 +1,169 @@
+* Long range RF antenna
+
+`ruuvitag datasheet`_
+
+Supported Features
+==================
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| NVIC      | on-chip    | nested vectored      |
+|           |            | interrupt controller |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial port          |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTT       | on-chip    | console              |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF52 DK website`_ and `Nordic Semiconductor Infocenter`_
+for a complete list of nRF52 Development Kit board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED0 (red) = P0.17
+* LED1 (green) = P0.19
+
+Push buttons
+------------
+
+* BUTTON0 = SW1 = P0.13
+
+Pin descriptions
+----------------
+
+.. figure:: img/pinout.jpg
+     :width: 442px
+     :align: center
+     :alt: RUUVI Pinout
+
+* 2 = P0.29 = SPI_SCK
+* 3 = P0.28 = SPI_MISO
+* 10 = P0.04 = GPIO (can be used as a GPIO / ADC pin)
+* 11 = P0.05 = GPIO (can be used as a GPIO / ADC pin)
+* 12 = P0.25 = SPI_MOSI
+* 13 = P0.19 = LED2 (green) / GPIO (can be used as a GPIO pin but the LED will blink)
+* 14 = P0.17 = LED1 (red) / GPIO (can be used as a GPIO pin but the LED will blink)
+* 15 = P0.13 = Button / GPIO (can be used as a GPIO pin)
+* 16 = GND (Battery's negative contact)
+* 17 = Battery's positive contact
+* 18 = Battery's positive contact
+* 19 = SWDIO
+* 20 = SWDCLK
+* 21 = P0.18 = SWO / GPIO (can be used as a GPIO pin)
+* 22 = P0.21 = Reset / GPIO (can be used as a GPIO pin if no need to reset the device)
+* 23 = GND (Battery's negative contact)
+* 24 = P0.31 = GPIO (can be used as a GPIO / ADC pin)
+* 25 = P0.30 = GPIO (can be used as a GPIO / ADC pin)
+
+GPIO = General Purpose Input Output pin
+
+P1 = Standard 10-pin ARM Cortex debug connector (on RuuviTag Rev.B1-B5)
+
+* 1 = VDD
+* 2 = SWDIO
+* 3 = GND (Battery's negative contact)
+* 4 = SWDCLK
+* 5 = GND (Battery's negative contact)
+* 6 = SWO
+* 7 = No Connect
+* 8 = No Connect
+* 9 = GND (Battery's negative contact)
+* 10 = Reset
+
+P1 = TC2030 TagConnect (on RuuviTag Rev.B6)
+
+* 1 = Battery's positive contact
+* 2 = SWDIO
+* 3 = Reset
+* 4 = SWDCLK
+* 5 = GND (Battery's negative contact)
+* 6 = SWO
+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+nrfjprog --family nrf52 --eraseall
```

Comment:
Where does the ``firmware.hex`` file come from?

This looks like command line steps for flashing.  If so, use a code block directive:

    .. code-block:: console

       nrfjprog --family nrf52 --eraseall
       nrfjprog --family nrf52 --program firmware.hex
       nrfjprog --family nrf52 --reset